src/audio/google/google_ctc_audio_processing.c

```diff
@@ -80,6 +79,12 @@ static void ctc_s16_default(struct google_ctc_audio_processing_comp_data *cd,
 	int samples_to_written = MIN(samples, audio_stream_samples_without_wrap_s16(sink, dest));
 	int written_samples = 0;
 
+	if (!cd->enabled) {
```

Comment:
this should be split into a different commit and using the switch control

Reply:
Split it into another commit now. Thanks!